addons/account/models/account_move.py

```diff
@@ -1785,7 +1785,7 @@ def unlink(self):
         for move in self:
             if move.name != '/' and not self._context.get('force_delete'):
                 raise UserError(_("You cannot delete an entry which has been posted once."))
-            move.line_ids.unlink()
+        self.line_ids.unlink()
```

Comment:
```suggestion
        self.mapped('line_ids').unlink()
```
AFAIK the mapped is not needed from v14 and forwards

Reply:
AFAIK it's since v13
@william-andre ?

Reply:
It is from v13 (along with all the ORM changes at that time)

And you can be sure that `unlink` is tested on runbot...

Reply:
Also, it was working without any issue with `move.line_ids.unlink()`, so why wouldn't it with `self` ?